app/models/organization_resolver.rb

```diff
@@ -48,6 +49,25 @@ def organization_tag_names
     @organization&.tags || []
   end
 
+  def entities
+    [
+      @organization,
+      @email,
+      @location,
+      @organization_name,
+      @phone_number,
+      @website
+    ]
+  end
+
+  def source_names
+    entities.reduce([]) do |memo, entity|
+      next entity if entity.nil?
+      name = entity.versions.last.actor.import.source.name
+      memo.include?(name) ? memo : memo << name
+    end
```

Comment:
I have a thought here - I'm thinking the organization should be able to tell us all of the sources that have contributed to it's various rankable data points. One thing that's missing from the implementation in the PR is that only sources of the highest rank are returned, but we want them all.

So yeah, I don't usually make a suggestion like this, but I think we should push this up to the model. 😨 

I think that would be better too because it would open up a sensible space for more unit tests.

Also note, we'd want `versions.first.actor` - def __not__ `last`.